src/kixi/gwyn/gwyn.clj

```diff
@@ -39,19 +40,53 @@
    (-> fire-station-lookup-table
        (wds/select-from-ds {:station fire-station})
        (wds/subset-ds :cols [:radius :lat :long]))})
+
+(def google-places-api "https://maps.googleapis.com/maps/api/place/nearbysearch/json?location=")
+
+(defn get-env-var [key]
+  (get (System/getenv) "places_api_key"))
+
+(defn api-str [loc k]
+  (str google-places-api loc "&key=" (get-env-var k)))
+
+(defn fetch-url [api-call]
+  (try (slurp api-call)
+       (catch Exception e (str "Fire station does not exist. Incorrect URL"))))
```

Comment:
Returning a string here will break `parse-google-places` because it'll attempt to json parse. Either re-throw an Exception with your own message (which should include the original exception message) or change `parse-google-places` so it's more tolerant of handling errors.